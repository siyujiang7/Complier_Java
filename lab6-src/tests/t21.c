long f(long a, char b, long c, char d) {
  printf("expecting: 5 a 6 b\n");
  printf("%ld\n", a);
  printf("%c\n", b);
  printf("%ld\n", c);
  printf("%c\n", d);
}

long main() {
  long x;
  x = 5;
  
  f(x, 'a', x + 1, 'b');
}
