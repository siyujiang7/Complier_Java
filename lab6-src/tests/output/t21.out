expecting: 5 a 6 b
5
a
6
b
